internal/filter/testcases/testcases.go

```diff
@@ -0,0 +1,111 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package testcases contains lists of ticket filtering test cases.
+package testcases
+
+import (
+	"math"
+
+	structpb "github.com/golang/protobuf/ptypes/struct"
+	"open-match.dev/open-match/internal/pb"
+)
+
+// TestCase defines a single filtering test case to run.
+type TestCase struct {
```

Comment:
I would prefer to move this file to `internal/filter` directory and declare it as a private type. What do you think?

Reply:
That's how I first made it, but I've intentionally moved it out.  Updated commit comment for more info.